frontend/test/resolution/testCallInitDeinit.cpp

```diff
@@ -1815,6 +1815,26 @@ static void test23d() {
       });
 }
 
+// Test if var declaration
+static void test24() {
+  testActions("test24",
+    R""""(
+      module M {
+        class R { }
+        proc test() {
+          var y : unmanaged R? = new unmanaged R();
```

Comment:
can you do `new unmanaged R?()`?

Reply:
sure, any particular reason?

Reply:
just simpler / gets you a smaller test. You don't really have to or anything.

Reply:
Went ahead and did this here as well as in those previous tests